utils/cq/d_logging_check.py

```diff
@@ -132,9 +132,11 @@ def __next__(self):
 # Logging macros where a new-line is always added.
 PREFIXES_NNL = ['DFUSE_LOG_WARNING', 'DFUSE_LOG_ERROR', 'DFUSE_LOG_DEBUG', 'DFUSE_LOG_INFO',
                 'DFUSE_TRA_WARNING', 'DFUSE_TRA_ERROR', 'DFUSE_TRA_DEBUG', 'DFUSE_TRA_INFO',
-                'DH_PERROR_SYS', 'DH_PERROR_DER',
-                'DL_ERROR', 'DHL_ERROR', 'DHL_WARN', 'DL_WARN', 'DL_INFO', 'DHL_INFO']
+                'DH_PERROR_SYS', 'DH_PERROR_DER']
 
+for prefix in ['DL', 'DHL', 'DS', 'DHS']:
```

Comment:
What does each prefx stand for? Does that need to be documented somewhere so people know what they're using?

Reply:
I don't want to make it too easy.  `H` is for handle, `S` is for system.

I'm working on a update to the coding style doc and will include this there.